test/controller_test.rb

```diff
@@ -24,6 +24,8 @@ def create
   def post(action, options = {})
```

Comment:
this is changing the method signature, so I think the if should be around different  `def`s

Reply:
... but I guess does not matter since it's just for our tests